csrankings-j.csv

```diff
@@ -403,7 +403,7 @@ Jan van Leeuwen,Utrecht University,https://www.uu.nl/staff/JvanLeeuwen1,7s_pSa4A
 Jan-J. Rückmann,University of Bergen,https://www.uib.no/en/persons/Jan-Joachim.R%C3%BCckmann,NOSCHOLARPAGE
 Jan-Michael Frahm,University of North Carolina,http://frahm.web.unc.edu,3YOe4NMAAAAJ
 Jan-Philipp Steghöfer,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/jan-philipp-steghofer.aspx,NOSCHOLARPAGE
-Jan-Willem van de Meent,Northeastern University,http://www.robots.ox.ac.uk/~jwvdm,CX9Lu38AAAAJ
+Jan-Willem van de Meent,Northeastern University,https://www.ccs.neu.edu/home/jwvdm/,CX9Lu38AAAAJ

```

Comment:
In fact, should change affiliation: "As of 1 September, I will start at the University of Amsterdam and will be on leave from Northeastern University.". Please update accordingly.

Reply:
Updated Jan-Willem's affiliation to the University of Amsterdam